Philips.CodeAnalysis.MaintainabilityAnalyzers/Maintainability/AvoidAsyncVoidAnalyzer.cs

```diff
@@ -45,7 +45,8 @@ private void Analyze(SyntaxNodeAnalysisContext context)
 			MethodDeclarationSyntax methodDeclaration = (MethodDeclarationSyntax)context.Node;
 
 
-			if (!methodDeclaration.Modifiers.Any(SyntaxKind.AsyncKeyword))
+			if (!methodDeclaration.Modifiers.Any(SyntaxKind.AsyncKeyword) ||
```

Comment:
You don't need to hardcode "void".  There's a way to compare with SyntaxKind.VoidKeyword.